src/instance/sql_instance.go

```diff
@@ -30,7 +30,7 @@ func CreateInstanceEntity(i *integration.Integration, con *connection.SQLConnect
 
 	if instanceRows[0].Name.Valid {
 		instanceNameIDAttr := integration.NewIDAttribute("instance", instanceRows[0].Name.String)
-		return i.EntityReportedVia(con.Host, con.Host, "ms-instance", instanceNameIDAttr)
+		return i.EntityReportedVia(con.Host, instanceRows[0].Name.String, "ms-instance", instanceNameIDAttr)
```

Comment:
If we're going to go back to naming the entity with the instance name, we'll need to replace the `instanceNameIDAttr` with an ID Attribute for host name to ensure the entity key is still unique